docs/source/reference/faq.rst

```diff
@@ -75,6 +75,20 @@ If you have edited the ``file_mounts`` section (e.g., by adding some files) and
 
 To avoid rerunning the ``setup`` commands, pass the ``--no-setup`` flag to ``sky launch``.
 
+How can I launch a VS Code tunnel using a SkyPilot task definition?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To launch a VS Code tunnel using a SkyPilot task definition, you can use the following task definition:
+
+.. code-block:: yaml
+    
+    setup: |
+      sudo snap install --classic code
```

Comment:
Should we add a comment like `# Supported on the default images of AWS, ...`?

I tried GCP and the image doesn't have snap. Getting
```
****** START Last lines of setup output ******
sudo: snap: command not found
******* END Last lines of setup output *******
```
It should have `apt`.

Reply:
I added a workaround in the comments. I just tested it on GCP and it works.